flow-server/src/main/java/com/vaadin/flow/server/VaadinServletService.java

```diff
@@ -98,8 +99,17 @@ protected VaadinServletService() {
         }
 
         if (getDeploymentConfiguration().enableDevServer()) {
-            DevModeHandlerManager.getDevModeHandler(this)
-                    .ifPresent(handlers::add);
+            Optional<DevModeHandler> handlerManager = DevModeHandlerManager
+                    .getDevModeHandler(this);
+            if (handlerManager.isPresent()) {
+                handlers.add(handlerManager.get());
+            } else {
+                getLogger()
+                        .warn("no DevModeHandlerManager implementation found "
+                                + "but dev server enabled. Include the "
+                                + "com.vaadin.vaadin-dev-server maven "
```

Comment:
```suggestion
                                + "com.vaadin:vaadin-dev-server:${vaadin.version} "
```

I think maven is redundant in this case, as gradle has this dependency as well. The syntax for the property was adopted from maven and matches the current property defined by start.vaadin.com.

Reply:
Reworded.

Reply:
(Skipped the version info because it is different from platform and bom-managed)